src/main/java/bio/terra/cda/app/service/QueryService.java

```diff
@@ -232,4 +241,24 @@ public String startQuery(String query) {
 
     return queryJob.getJobId().getJob();
   }
+
+  /**
```

Comment:
In general I would avoid committing commented out code, for many reasons. You can keep the code locally until it's ready to be used.

Reply:
fixed

Reply:
I don't see any changes for this